resources/definitions/adimlab_gantry.def.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "ADIMLab Gantry",
+    "version": 2,
+    "inherits": "adimlab_base",
```

Comment:
If there is only one machine that uses this base, merge it into a single definition.